omod/src/main/resources/config.xml

```diff
@@ -89,6 +89,14 @@
 			Format used by Javascript functions for dates that have a time component
 		</description>
 	</globalProperty>
+
+	//This GP should be removed once the timezone.conversions GP is provided by Core, UIFR depends from core.
```

Comment:
TODO?

Reply:
Yes, that is something to do in the future, I added `TODO` on the comments.